pkg/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"errors"
+	"os"
+	"path"
+)
+
+func GetConfigDir() (string, error) {
+	confDir := path.Join(os.Getenv("HOME"), ".config", "thyra")
+	if _, err := os.Stat(confDir); os.IsNotExist(err) {
```

Comment:
it looks like there are 2 instructions at the same line

`_, err := os.Stat(confDir)`

and 

`os.IsNotExist(err)`

which could be a bad practice.

moreover, I don't understand what the if compares, I guess it's the truthiness of the second instruction. It looks strange to me as a no golang dev

Reply:
the second instruction checks if the error is of type 'no exist error'. but yes you are right, i will just check for any error